tests/integration/test_register_persister.py

```diff
@@ -53,3 +68,22 @@ def test_load_cassette_with_custom_persister(tmpdir, httpbin):
     with my_vcr.use_cassette(test_fixture, serializer="json"):
         response = urlopen(httpbin.url).read()
         assert b"difficult sometimes" in response
+
+
+def test_load_cassette_with_custom_persister(tmpdir, httpbin):
+    """
+    Ensure expected errors from persister are swallowed while unexpected ones
+    are passed up the stack.
```

Comment:
Could you make "stack" be "call stack"? I think it would make it a tiny bit more clear.

Reply:
Done!